core/domain/user_domain.py

```diff
@@ -797,6 +797,31 @@ def remove_learnt_topic_id(self, learnt_topic_id):
         self.learnt_topic_ids.remove(learnt_topic_id)
 
 
+class LearnerGoals(python_utils.OBJECT):
+    """Domain object for the learner goals model."""
+
+    def __init__(
+            self, user_id, topic_ids_to_learn):
+        self.id = user_id
+        self.topic_ids_to_learn = topic_ids_to_learn
+
+    def add_topic_id_to_learn(self, topic_id):
+        """Adds the topic id to learn list.
```

Comment:
Adds a new topic ID to the 'topic IDs to learn' list.

Reply:
Done.